### DISCUSSION TITLE
Ipopt on Jetson modules (arm arch in general)

### DISCUSSION THREAD

**Discussion by winwinashwin** [coin-or/Ipopt#495] [Q&A] [answered]
Hello! I am quite new to the Ipopt community. I am trying to deploy a model predictive control system using Ipopt as the solver on a Nvidia Jetson module. Does Ipopt support arm based cpus? If not are there any alternatives??

**Comment by svigerske** [comment]
ARM based CPUs should work in general (There is travis test that builds Ipopt on Ubuntu on ARM64: https://travis-ci.com/github/coin-or/Ipopt/jobs/514129657). Whether that works in your exact setting, I don't know. Essentially, you'll need C/C++ and probably also Fortran compilers.

**Comment by damienhocking** [comment, marked as answer]
Hi, this works, we use IPOPT on Jetsons all the time. It's a regular Linux build, we use vanilla lapack and MUMPS.

Damien
